### DISCUSSION TITLE
스피커 앱, 리스너 앱 로딩 다이얼로그 관련

### DISCUSSION THREAD

**Discussion by Jsueeee** [euphony-io/check-in-listener#32] [General]
이..걸... 유포니쪽에 올려야하나 어디에 올려야하나 고민했는데 일단 저희 라이브러리에 올려보겠습니다!
로딩쪽을 맞추는 게 좋을 것 같은데 피그마에 있던 로딩다이얼로그 넣으셨을까요?
@jopopcorn  조수연님! 혹시 멘토님이 말씀해주셨던 요거 쓰셨을까요??
https://github.com/ybq/Android-SpinKit

**Comment by jopopcorn** [comment]
아하 프로그레스바 말씀하시는거죠?!
피그마의 디자인은 그냥 임의로 로딩 아이콘을 넣었고요 xml에는 안드로이드 프로그레스바 위젯 추가해서 체크인 시작/종료에 따라 visibility 처리를 해주었습니다!

**Reply by Jsueeee** [reply]
혹시 바꾸셔도 괜찮으시면 Fading Circle 어떠세요??

**Reply by jopopcorn** [reply]
> 혹시 바꾸셔도 괜찮으시면 Fading Circle 어떠세요??

앗 저는 좋습니다 ㅎㅎ 그럼 추가로 라이브러리 적용해서 변경해보겠습니다!

**Reply by Jsueeee** [reply]
감사합니다 ㅎㅎ
리스너앱은 수신 완료 하면 
https://lottiefiles.com/67000-checkmark
이런 거로 표시하는 것도 생각하고 있습니다!

**Reply by Jsueeee** [reply]
그리고 저희 앱 아이콘도 맞추면 좋을 것 같아요!
그냥 초록색 아이콘 보다는 무료아이콘이라도 맞춰서 넣는 게 보기 좋을 것 같습니다 ㅎㅎㅎ

**Reply by jopopcorn** [reply]
오호 로티 적용하시는군요 애니메이션 있는게 훨씬 좋아보이네요!
스피커에서는 수신 완료를 확인할 수 있는 방법이 없어서 아쉽네요 ㅜㅜ 리스너쪽에서는 확실히 표시하는 게 좋을 것 같습니다!

**Reply by jopopcorn** [reply]
> 그리고 저희 앱 아이콘도 맞추면 좋을 것 같아요! 그냥 초록색 아이콘 보다는 무료아이콘이라도 맞춰서 넣는 게 보기 좋을 것 같습니다 ㅎㅎㅎ

넵넵 좋습니다ㅋㅋㅋ 아니면 유포니 로고를 아이콘으로 지정하는 게 좋을까요? 어떤걸로 적용해야 일관성 있을지도 고민되네요..!

**Reply by Jsueeee** [reply]
> 오호 로티 적용하시는군요 애니메이션 있는게 훨씬 좋아보이네요! 스피커에서는 수신 완료를 확인할 수 있는 방법이 없어서 아쉽네요 ㅜㅜ 리스너쪽에서는 확실히 표시하는 게 좋을 것 같습니다!

아 스피커쪽에서 적용하기는 어렵군요ㅠㅠㅠ 넵 감사합니다!

**Reply by Jsueeee** [reply]
> > 그리고 저희 앱 아이콘도 맞추면 좋을 것 같아요! 그냥 초록색 아이콘 보다는 무료아이콘이라도 맞춰서 넣는 게 보기 좋을 것 같습니다 ㅎㅎㅎ
> 
> 넵넵 좋습니다ㅋㅋㅋ 아니면 유포니 로고를 아이콘으로 지정하는 게 좋을까요? 어떤걸로 적용해야 일관성 있을지도 고민되네요..!

오 유포니 로고는 생각도 못했어요 !! 제일 좋을 것 같긴 한데 스피커리스너 둘 다 같은 건 좀 아닌 것 같구... 컬러 변경??? 어떨까요! 지금 파란 컬러 이용해서 
하얀배경 파란 로고 / 파란배경 하얀 로고
이렇게 해도 좋을 것 같아요!!

**Reply by jopopcorn** [reply]
> > > 그리고 저희 앱 아이콘도 맞추면 좋을 것 같아요! 그냥 초록색 아이콘 보다는 무료아이콘이라도 맞춰서 넣는 게 보기 좋을 것 같습니다 ㅎㅎㅎ
> > 
> > 
> > 넵넵 좋습니다ㅋㅋㅋ 아니면 유포니 로고를 아이콘으로 지정하는 게 좋을까요? 어떤걸로 적용해야 일관성 있을지도 고민되네요..!
> 
> 오 유포니 로고는 생각도 못했어요 !! 제일 좋을 것 같긴 한데 스피커리스너 둘 다 같은 건 좀 아닌 것 같구... 컬러 변경??? 어떨까요! 지금 파란 컬러 이용해서 하얀배경 파란 로고 / 파란배경 하얀 로고 이렇게 해도 좋을 것 같아요!!

오호 괜찮을 것 같아요! 근데 직접 작업을 해야할 것 같은데 혹시 포토샵이 있으신가요??
저는 포토샵이 없어서 피그마로 아이콘에 뒷배경 정도만 작업할 수 있을 것 같아요 ㅜㅜ

**Reply by jopopcorn** [reply]
<img width="505" alt="스크린샷 2021-10-03 오전 12 46 07" src="https://user-images.githubusercontent.com/35393459/135723501-aab82fda-b03c-44bb-bde2-0f4a37197e6c.png">
뒷배경만 제거하고 동그란 모서리 배경으로 바꿔봤는데 요건 어떠신가요?!

**Reply by Jsueeee** [reply]
> <img alt="스크린샷 2021-10-03 오전 12 46 07" width="505" src="https://user-images.githubusercontent.com/35393459/135723501-aab82fda-b03c-44bb-bde2-0f4a37197e6c.png">
> 
> 뒷배경만 제거하고 동그란 모서리 배경으로 바꿔봤는데 요건 어떠신가요?!

오 충분한 것 같습니다!! 능력자 👍👍👍👍

**Reply by Jsueeee** [reply]
@jopopcorn 혹시 저희 쪽에도 이미지 파일 공유 부탁드려도 될까요? :)

**Reply by jopopcorn** [reply]
@Jsueeee @psh2849 
넵! 60, 108 사이즈별로 로고 이미지 만들었습니다!!
두 분 다 [피그마](https://www.figma.com/file/0e7hUwdXmSIhoJmresxEQj/Euphony-Check-in-Speaker?node-id=87%3A2)에서 edit 권한을 드렸는데 혹시 export 가능하면 이미지 클릭 후 오른쪽 맨아래 export 메뉴에서 다운로드 받으시면 될 것 같아요!!
안된다면 구글 드라이브로 공유드리겠습니다!

**Reply by jopopcorn** [reply]
로고는 파란배경 -> 스피커, 하얀배경 -> 리스너로 할까요?!

**Reply by Jsueeee** [reply]
> 로고는 파란배경 -> 스피커, 하얀배경 -> 리스너로 할까요?!

넵 좋습니다 😁

**Reply by Jsueeee** [reply]
> @Jsueeee @psh2849 넵! 60, 108 사이즈별로 로고 이미지 만들었습니다!! 두 분 다 [피그마](https://www.figma.com/file/0e7hUwdXmSIhoJmresxEQj/Euphony-Check-in-Speaker?node-id=87%3A2)에서 edit 권한을 드렸는데 혹시 export 가능하면 이미지 클릭 후 오른쪽 맨아래 export 메뉴에서 다운로드 받으시면 될 것 같아요!! 안된다면 구글 드라이브로 공유드리겠습니다!

해봤는데 export 가능하네요! 감사합니다 😆😆😆

**Comment by psh2849** [comment]
저 링크에 있는 다이얼로그들 다 괜찮네요 오..

**Comment by psh2849** [comment]
말씀해주신 Fading Circle로 적용을 시켜 봤습니다!
결과는 https://github.com/euphony-io/check-in-speaker/issues/29#issuecomment-932782533 요기서 보실 수 있습니다!

다이얼로그가 색상을 지정해줘야 제대로 출력이 되더라구요
그냥 기본 색인 검은색으로 할지 아니면 다른 색을 생각 중 이신게 있는지 여쭤보려고 왔습니다!

**Reply by dlwlstks96** [reply]
오오 대단하십니다...!!👍👍

**Reply by jopopcorn** [reply]
@Jsueeee 다이얼로그 색상은 어떤걸로 할지 답변해주심 감사하겠습니다~!

**Reply by Jsueeee** [reply]
> @Jsueeee 다이얼로그 색상은 어떤걸로 할지 답변해주심 감사하겠습니다~!

화면 안에 있는 것 중에,
개인안심번호박스의 하늘색이나 버튼 파란색 둘 중에 하나 골라서 통일하는 게 좋을 것 같습니다!

**Reply by jopopcorn** [reply]
> > @Jsueeee 다이얼로그 색상은 어떤걸로 할지 답변해주심 감사하겠습니다~!
> 
> 화면 안에 있는 것 중에, 개인안심번호박스의 하늘색이나 버튼 파란색 둘 중에 하나 골라서 통일하는 게 좋을 것 같습니다!

넵넵 그게 좋겠네요! 흐릿한 것보다는 잘 보이는 게 좋을 것 같으니 버튼의 파란 색상으로 해도 좋을 것 같아요!!

**Reply by Jsueeee** [reply]
좋아요~ 🙆‍♀️
